pkg/controller/instance/apply_conventions.go

```diff
@@ -25,22 +25,20 @@ import (
 
 const basePath = "/kustomize"
 
-// metadata contains metadata associated with current PlanExecution
+// metadata contains metadata associated with current plan being executed
 type metadata struct {
```

Comment:
You could also make `metadata` exportable since we would use it in the tasker branch but that's a soft nit